datadog_lambda/constants.py

```diff
@@ -42,3 +42,11 @@ class XrayDaemon(object):
     XRAY_TRACE_ID_HEADER_NAME = "_X_AMZN_TRACE_ID"
     XRAY_DAEMON_ADDRESS = "AWS_XRAY_DAEMON_ADDRESS"
     FUNCTION_NAME_HEADER_NAME = "AWS_LAMBDA_FUNCTION_NAME"
+
+
+class Headers(object):
+    Parent_Span_Finish_Time = "x-datadog-parent-span-finish-time"
```

Comment:
Can you add a comment here and to `Authorizing_Request_Id` explaining why we need each?